mkt/regions/migrations/0001_initial.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+
+from mkt.constants import regions
+from mkt.developers.cron import exclude_new_region
+
+
+def exclude_romania(apps, schema_editor):
+    exclude_new_region([regions.RO])
+
+
+class Migration(migrations.Migration):
+    operations = [
+        migrations.RunPython(exclude_romania),
```

Comment:
Oh, and we should consider making `exclude_new_region` migration-aware if we are going to use it in django migrations in the future. Right now importing it imports a whole lot of other stuff and it uses the django model directly, which is a big no-no. It'd probably break if another migration modifies `Webapp` and someone tries to run both migrations in the same run.